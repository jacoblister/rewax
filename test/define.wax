(@define N 8)

(func define_value_test
    (print "define_value_test:")
    (print @N)
)

(func main (result int)
    (call define_value_test)
    (return 0)
 )
