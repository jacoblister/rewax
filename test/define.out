define_value_test:
8
